assets/sass/_split-section.scss

```diff
@@ -0,0 +1,41 @@
+.split-section {
+
+    &--container {
+        @apply pt-8 px-0;
+    }
+
+    &--left {
+        @apply pr-8;
+    }
+
+    &--right {
+        @apply pl-8;
+    }
+
+    &--cta-btn {
+        @apply mt-8 text-xl;
+    }
+
+    @media (max-width: 768px) {
+        &--left {
+            @apply pr-0;
+        }
+
+        &--right {
+            @apply pl-0;
+        }
+
+        &--cta-btn {
+            @apply w-full text-center;
+        }
+    }
+
+    &--container {
```

Comment:
By and large, we don't use BEM notation like this; instead, we follow the Tailwind convention of duplicating the class name in order to get the compositional behavior -- so `btn btn-cta`, for example, as opposed to `btn--cta`.